components/pedigree/resources/src/main/resources/pedigree/view/personGroupVisuals.js

```diff
@@ -26,14 +26,6 @@ define([
             this.setNumPersons(node.getNumPersons());
         },
 
-        generateHoverbox: function(x, y) {
-            if (editor.isReadOnlyMode()) {
-                return new ReadOnlyHoverbox(this.getNode(), x, y, this.getGenderGraphics());
-            } else {
-                return new PersonGroupHoverbox(this.getNode(), x, y, this.getGenderGraphics());
```

Comment:
@allasm I better just remove this separate [misc] commit, it's not related to the issue anyways.